### ISSUE TITLE
Any hope of having pretrained weights ?

### ISSUE THREAD

**Issue by 0xsimulacra** [QiaoranC/tf_ResNeSt_RegNet_model#5]
Hello, 

Thanks for sharing your ready to train code, I was wondering if there any hope to have the pretrained weights on Image-Net ? I can contribute by adding Cascade-RCNN and Faster-RCNN to the model as having these pretrained weights will help me to test the performance of the model on a task without the need to learn Pytorch as I don't have eanough time.

If there no chance, how long do you think it will take me to train the model (Resnest-200) on Colab pro ?

Thank you.

**Comment by QiaoranC** [comment]
Hi, the pertrain model is indeed a reasonable ask, but i dont have much time/power/gpu to do it, Sorry for that. And welcome to contribute it.
As for the time of Colab pro, sorry i never use them, cannt give you any suggestions.
I leave this open. And again, welcome anyone contribute the pertrain model or the usage feedback or improvements/suggests.

**Comment by 0xsimulacra** [comment]
Thank you for the clarification, I assumed you just didn't want to release the weights. I too have only the limited power of Colab Pro (1 x 16GB GPU) and the other problem is that I'm not that experienced to pretrain a model on Benchmark dataset. If I find I can do it (in the required power side and time/reading needed to program the learning rate scheduleer, the data processing and the training strategy) I'll let you know.
